CIP-0013/CIP-0013.md

```diff
@@ -1,27 +1,37 @@
 ---
 CIP: 13
 Title: Cardano URI Scheme
-Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>
+Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>, Robert Phair <rphair@cosd.com>
 Comments-URI:
 - https://github.com/Emurgo/EmIPs/pull/2
 - https://forum.cardano.org/t/cip-cardano-payment-uri-scheme/41457
+- https://github.com/cardano-foundation/CIPs/pull/25
+- https://github.com/cardano-foundation/CIPs/pull/61
+- https://forum.cardano.org/t/cip-stake-uri-scheme-for-pools-delegation-portfolios/40594
 Status: Draft
 Type: Informational
-Created: 2020-10-20
+Created: 2020-09-22
 License: CC-BY-4.0
 ---
 
 # Abstract
 
-This proposal describes a basic URI scheme to handle ADA transfers, as well
-as possible approaches for a multiplatform implementation.
+This proposal describes a basic URI scheme to handle ADA transfers and links to stake pools, as well as possible approaches for a multiplatform implementation.
 
 # Motivation
 
+#### For payment URIs: 
+
 Users who create community content often want donations as a financial incentive. However, forcing users to open their wallet and copy-paste an address lowers the amount of people likely to send tokens (especially if they have to sync their wallet first).
 
 If donating was as simple as clicking a link that opens a light wallet with pre-populated fields, users may be more willing to send tokens. URI schemes would enable users to easily make payments by simply clicking links on webpages or scanning QR Codes.
 
+#### For stake pool URIs: 
+
+Centralised sources of information, such as the Daedalus ranking algorithm and its presentation to the delegating user, have led a growing amount of stake to be disproportionately assigned to pools pushed near & beyond the saturation point.
```

Comment:
IMO this sounds a bit harsh/unfair on Daedalus, especially because "centralised" has a bad connotation in the industry. The problem, in general, is simply that users tend to make their delegation choices following only a limited set of information sources —typically those provided by wallet interfaces or blockchain data aggregation sites— but is not a problem with Daedalus per se.

Also

> a convenient and popular alternative to the trend of stake centralisation

Being a bit picky, I don't thing this CIP is a direct solution to the problem of stake centralisation. Rather, it would just help us mitigate the problem by facilitating delegation transactions.

Reply:
1. removed phrase "such as the Daedalus ranking algorithm and its presentation to the delegating user"
2. changed "convenient and popular" to "possible"